AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -4,6 +4,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38487
 sncf.com#@#.cookie-footer
+! https://github.com/AdguardTeam/AdguardFilters/issues/37957
+!+ PLATFORM(ios, ext_android_cb)
```

Comment:
Please remove the platform elements, I could reproduce on mobile view on my desktop device. The cookie entries were already created before, but was stuck on black screen.